spec/controllers/v1.0/requests_controller_spec.rb

```diff
@@ -219,48 +228,57 @@
     end
   end
 
-  describe 'Accessible requests for Approver persona' do
-    let(:ctrl) { described_class.new }
-    let(:group_a) { double(:name => 'group_a', :uuid => "g_a") }
-    let(:group_b) { double(:name => 'group_b', :uuid => "g_b") }
-    let(:group_c) { double(:name => 'group_c', :uuid => "g_c") }
-    let(:group_d) { double(:name => 'group_d', :uuid => "g_d") }
-    let(:group_e) { double(:name => 'group_e', :uuid => "g_e") }
-    let!(:workflow_a) { create(:workflow, :name => 'workflow_a', :group_refs => [group_a.uuid, group_b.uuid, group_c.uuid]) }
-    let!(:workflow_b) { create(:workflow, :name => 'workflow_b', :group_refs => [group_b.uuid, group_d.uuid, group_e.uuid]) }
-    let!(:approver_request1) { create(:request, :workflow_id => workflow_a.id, :group_ref => group_a.uuid, :tenant_id => tenant.id) }
-    let!(:approver_request2) { create(:request, :workflow_id => workflow_b.id, :group_ref => group_b.uuid, :state => 'notified', :tenant_id => tenant.id) }
+  describe 'Accessible requests for approvers or requesters' do
+    let(:template) { create(:template) }
+    let(:group_a) { instance_double(Group, :name => 'group_a', :uuid => "g_a") }
```

Comment:
@bzwei Should this be the RBACApiClient::GroupOut instead of Group?

Reply:
It is `Group`. We have a `Group` class that wraps `RBACApiClient::GroupOut`